opencog/combo/converter/combo_atomese.cc

```diff
@@ -45,7 +45,7 @@ Handle ComboToAtomeseConverter::atomese_argument(const argument &a,
 	switch (parent_procedure_type) {
 		case id::predicate: {
 			Handle tmp = createNode(PREDICATE_NODE,
-			                        to_string(a.is_negated() ? -a.idx : a.idx));
+			                        to_string(a.is_negated() ? -a.idx : a.idx)+_suffix);
```

Comment:
According to the standard https://wiki.opencog.org/w/Development_standards#Formatting you need to put whitespaces at the lest and right of `+`.

Reply:
And keep the line contained within 80 columns.